src/editor/controller/editor_controller.cpp

```diff
@@ -130,8 +132,7 @@ void editor_controller::init_tods(const game_config_view& game_config)
 		tods_map::iterator times = tods_.find(schedule_id);
 		if (times == tods_.end()) {
 			std::pair<tods_map::iterator, bool> new_times =
-				tods_.emplace(schedule_id, std::pair(schedule_name, std::vector<time_of_day>()));
-
+			tods_.emplace(schedule_id, std::pair(schedule_name, std::vector<time_of_day>()));
```

Comment:
This looks like an accidental change?

Reply:
Fixed (8bcca43af0f6bcc3eaf9060940e54708a60c5159).